Extension/i18n/cht/package.i18n.json

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 // Do not edit this file. It is machine generated.
 {
-	"c_cpp.capabilities.untrustedWorkspaces.description": "多個設定可能會導致工作區的本機進程得以執行，例如 C_Cpp. clang_format_path、C_Cpp addNodeAddonIncludePaths、C_Cpp. compilerPath、C_Cpp.default.configurationProvider 以及 C_Cpp. compileCommands 以及 c_cpp_properties.js 上的對等屬性。",
```

Comment:
This is a new regression bug -- can you open a bug on the loc team?

Reply:
@Colengms FYI, my clang-tidy branch has these settings strings changed to markdown like `#C_Cpp.<setting>#` so that could make it more obvious to the loc team that they shouldn't be changing it.